index.js

```diff
@@ -61,7 +61,12 @@ const options = {
       assertEndpoint: process.env.CORPPASS_ASSERT_ENDPOINT,
     },
   },
-  showLoginPage: process.env.SHOW_LOGIN_PAGE === 'true',
+  showLoginPage: (req) => {
+    if (req.header('X-Show-Login-Page')) {
+      return req.header('X-Show-Login-Page') === 'true'
+    }
+    return process.env.SHOW_LOGIN_PAGE === 'true'
```

Comment:
How about something more concise like..
```suggestion
    return process.env.SHOW_LOGIN_PAGE === 'true' || req.header('X-Show-Login-Page') === 'true'
```

Reply:
It was actually to support a possible use case where the `SHOW_LOGIN_PAGE` env is true but the user wants to override it to false in a request via a header

Reply:
I considered this form, but thought it was quite hard to read:
```js
showLoginPage: (req) =>
    req.header('X-Show-Login-Page')
      ? req.header('X-Show-Login-Page') === 'true'
      : process.env.SHOW_LOGIN_PAGE === 'true',
```

Reply:
Oh we could also do something like this:
```js
showLoginPage: (req) =>
    (req.header('X-Show-Login-Page') || process.env.SHOW_LOGIN_PAGE) === 'true',
```